src/styles/global.css

```diff
@@ -43,6 +43,9 @@ body {
   width: 100vw;
   background-color: var(--primary-bg);
 }
+main {
+  padding: 1rem;
```

Comment:
Do we really need this at the global level? This limits what we can do further down (e.g. display image at full page width).